statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1709,6 +1710,20 @@ def get_prediction(self, exog=None, exposure=None, offset=None,
 
     get_prediction.__doc__ = pred.get_prediction_glm.__doc__
 
+    def score_test(self, exog_extra=None, params_constrained=None,
+                   hypothesis='joint', cov_type=None, cov_kwds=None,
+                   k_constraints=None, observed=True):
+
+        res = infer.score_test(self, exog_extra=exog_extra,
```

Comment:
the original GLM.score_test is not used anymore?
The new version allows for some robust sandwich forms